administration/live-migration.adoc

```diff
@@ -0,0 +1,138 @@
+# Live Migration
+
+Live migration is a process during which a running Virtual Machine Instance
+moves to another compute node while the guest workload continues to run and
+remain accessable.
+
+## Initiate live migration
+
+Live migration is initiated by posting a VirtualMachineInstanceMigration (VMIM)
+object to the cluster.
+The example below starts a migration process for a virtual machine instance
+`vmi-fedora`
+
+```yaml
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstanceMigration
+metadata:
+  name: migration-job
+spec:
+  vmiName: vmi-fedora
+configuration:
```

Comment:
Is this a top-level item? `VMIM.configuration`?

Reply:
fixed.